packages/components/ContextualMenu/src/ContextualMenu.types.ts

```diff
@@ -0,0 +1,33 @@
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { ICalloutProps, ICalloutTokens } from '@fluentui-react-native/callout';
+
+export const contextualMenuName = 'ContexualMenu';
+
+export type IContextualMenuTokens = ICalloutTokens;
+/**
+ * Properties and Tokens for FluentUI React Native ContextualMenu
+ */
+
+export interface IContextualMenuProps extends Omit<ICalloutProps, 'setInitialFocus'> {
+  /*
+  * Whether to set initial focus on the contextual menu container, as opposed to the first menu item.
+  */
+  shouldFocusOnContainer?: boolean;
+
+  /*
+  * Whether tofocus on the menu when mounted
+  */
+  shouldFocusOnMount?: boolean;
```

Comment:
So this is effectively a replacement for `setInititalFocus`? Am I right in guessing that shouldFocusOnMount is what fluent web currently uses?

Reply:
Yes shouldFocusOnMount is what fluent web currently uses. This might actually be worth a discussion. Callout supports setInitialFocus currently, and I'm now thinking we should setting on one for both ContextualMenu and Callout.

Reply:
jk I talked to Jon and we should keep them separate for each component

Reply:
@lenahong what's the intended difference?

Reply:
He says that ContextualMenu actually passes shouldFocusOnMount to Callout's setInitialFocus and they should get unified eventually, but for now they're different.